stock_barcodes_gs1/tests/test_stock_barcodes_gs1.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2108-2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.addons.stock_barcodes.tests.test_stock_barcodes import TestStockBarcodes
+
+
+@common.tagged("post_install", "-at_install")
+class TestStockBarcodesGS1(TestStockBarcodes, MailCommon):
+    # pylint: disable=W8121
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        gs1_nomenclature = cls.env.ref(
+            "barcodes_gs1_nomenclature.default_gs1_nomenclature"
+        )
+        # Odoo creash with default separator (Alt029|#|\x1D) so remove it from settings
+        gs1_nomenclature.gs1_separator_fnc1 = False
```

Comment:
I think you can't initialize the `gs1_separator_fnc1` field as `False` because it's a `Char` field. I think initializing it with an empty string (`""`) should work.